.gitattributes

```diff
@@ -1 +0,0 @@
-*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
I think this needs to stay, and you also need to add svg files to be handled by git lfs for this to work as intended.